### PR TITLE
Tekton Triggers 0.4 support for webhooks extension

### PR DESCRIPTION
**Pull request by a-roberts**
# Changes

Also makes the webhook name necessary to be unique.

For https://github.com/tektoncd/experimental/issues/493

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/experimental/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by a-roberts** [comment]
`initialFocus` did not return a node`

great

/test tekton-experimental-unit-tests

**Comment by a-roberts** [comment]
PR finally ready for a look please (I've just been retesting this repeatedly, definitely not working at this time 😄)

File: webhooks-extension/docs/Parameters.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ spec:
 
 # TriggerTemplate Parameters
 
+TODO CHECK ONCE PIPELINE HOTEL EXAMPLES WORK
```
Comment (dibbles) [inline webhooks-extension/docs/Parameters.md:37]: TODO - Is this done???

**Comment by a-roberts** [comment]
> there are quite a few places where we import v1alpha1 of pipelines .... is this release supposed to be for beta? ... if so shouldn't these imports change too?

Yeah that's an interesting one, I asked about that https://github.com/tektoncd/dashboard/issues/1218 here and Vincent mentions automatic conversion going on, will remove the Todo now thanks

**Comment by a-roberts** [comment]
Helm tests popping up so raised on Slack

**Comment by eddycharly** [comment]
@a-roberts `pull-tekton-experimental-helm-lint` is fixed now, sorry for the inconvenience

**Pull request merged**